pkgs/data/fonts/source-han-sans/default.nix

```diff
@@ -26,22 +31,26 @@ in
 {
   japanese = makePackage {
     language = "japanese";
-    region = "JP";
     description = "Japanese subset of an open source Pan-CJK typeface";
+    url = "https://github.com/adobe-fonts/source-han-sans/raw/5f5311e71cb628321cc0cffb51fb38d862b726aa/SubsetOTF/SourceHanSansJP.zip";
```

Comment:
Wouldn't it make sense to specify the revision as a variable. Otherwise you would have to edit four lines when updating the package.